src/components/Sidebar.js

```diff
@@ -141,11 +143,18 @@ export default ({
             )
           })
         ) : (
-          // Not sure if I should abstract this case to use links.map
           <Link href={'/application'}>
             <StyledA selected={location === '/application'}>
               <ApplicationText>APPLICATION</ApplicationText>
-              <StatusText>{hackerStatuses()[hackerStatus]?.sidebarText}</StatusText>
+              <StatusText>
+                {hackerStatus === 'acceptedNoResponseYet'
+                  ? !isLoadingAppStatus
+                    ? isRsvpOpen
+                      ? hackerStatuses()[hackerStatus]?.sidebarText
+                      : 'NO RSVP'
+                    : null
+                  : hackerStatuses()[hackerStatus]?.sidebarText}
+              </StatusText>
```

Comment:
This logic seems to be duplicating that of `components/ApplicationDashboard`. As well, `Sidebar` should only handle presentation logic of the UI. I recommend moving this logic to the parent container and adding an additional case handling no RSVP in the `hackerStatuses`